acceptance/acceptance_test.go

```diff
@@ -150,7 +150,8 @@ func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
 	repls.SetPath(execPath, "[CLI]")
 
 	// Make helper scripts available
-	t.Setenv("PATH", fmt.Sprintf("%s%c%s", filepath.Join(cwd, "bin"), os.PathListSeparator, os.Getenv("PATH")))
+	pathDirs := []string{filepath.Join(cwd, "bin"), buildDir}
+	t.Setenv("PATH", fmt.Sprintf("%s%c%s", strings.Join(pathDirs, string(os.PathListSeparator)), os.PathListSeparator, os.Getenv("PATH")))
```

Comment:
This adds buildDir to PATH? What for?

Reply:
If the goal is to expose dlt command to test then we should 

- modify BuildCLI to link databricks to dlt
- export $DTL env var pointing to built dlt (+ replacement)